app/src/main/java/com/topjohnwu/magisk/core/tasks/MagiskInstallImpl.kt

```diff
@@ -258,6 +264,10 @@ abstract class MagiskInstallImpl : FlashResultListener {
             return false
         }
 
+        if (!srcNand.isEmpty()) {
```

Comment:
This line will cause a crash in case the first if is not a success. You cannot access lateinit property when it's uninitialised. I'd suggest changing
`private lateinit var srcNand: String`
to
`private var srcNand: String = ""`

Reply:
Thanks! Fixed up per your suggestion in my latest force push. :raised_hands: